Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+﻿<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<Project>
+
+    <PropertyGroup Condition="'$(MSBuildProjectExtension)' == '.csproj'">
+        <Configurations>Debug;Release;Stable;Preview;Store</Configurations>
```

Comment:
The stable, preview, and store configurations were recently removed.

Reply:
I'll remove them soon